dev/VSIX/ProjectTemplates/Desktop/CppWinRT/UnitTestApp/readme.txt

```diff
@@ -0,0 +1,27 @@
+========================================================================
```

Comment:
looks like C++/WinRT app project template text - change to something more unittest appropriate?

Reply:
I've updated it. Once we get a doc page up we can update this readme to point to it.